tests/seek_stress.rs

```diff
@@ -69,19 +71,28 @@ use std::ffi::c_double;
 use std::ffi::c_float;
 use std::ffi::c_int;
 use std::ffi::c_uint;
+#[cfg(not(target_os = "windows"))]
 use std::ffi::c_ulonglong;
```

Comment:
For `cfg`ed `use`s, I put them separate from all of the other `use`s since they mess up `rustfmt` sorting the `use`s.  Not too important I guess since this is in the tools, but it's nicer this way generally.